redhat/README.rpm_build.md

```diff
@@ -29,7 +28,10 @@ Building your own FRRouting RPM
         cd frr
         ./bootstrap.sh
         ./configure --with-pkg-extra-version=-MyRPMVersion
+	On CentOS/RHEL:
         make SPHINXBUILD=sphinx-build2.7 dist
+	On Fedora:
+        make SPHINXBUILD=sphinx-build-3 dist
```

Comment:
Are you sure it's named `sphinx-build-3` and not `sphinx-build3`? The latter is rather common (and autodetected by `configure`.)

Reply:
Yes, on F27 and up at least
  
  $ which sphinx-
  sphinx-apidoc-3        sphinx-autogen-3       sphinx-build-3         sphinx-quickstart-3    
  sphinx-apidoc-3.6      sphinx-autogen-3.6     sphinx-build-3.6       sphinx-quickstart-3.6

Some other python3 package name mess with recent fedoras too, I'll do annother PR for that